packages/elements/src/components/MosaicTableOfContents/TableOfContents.tsx

```diff
@@ -24,6 +24,14 @@ import {
 const ActiveIdContext = React.createContext<string | undefined>(undefined);
 const LinkContext = React.createContext<CustomLinkComponent | undefined>(undefined);
 
+const NODE_TYPE_META_ICON: { [nodeType: string]: IIconProps['icon'] } = {
```

Comment:
Why those constants were moved here though? :thinking:

Reply:
There's no need to update Mosaic directly. What I did here was just moving the constants to Elements and using the `fas` prefix which indicates the free package from `FontAwesome`

Reply:
Mosaic constants use `fal`, which points to a Pro version. Elements doesn't handle the Pro version so it fails to display these icons

Reply:
I still don't understand. There is `./constants` import. This file keeps... well, constants, as the name suggests.

And yet you moved *some* constants from that `./constants` file to the main file, while you left some others.

I don't fully understand why.

Reply:
![Screenshot from 2021-04-30 15-00-10](https://user-images.githubusercontent.com/7916523/116698600-d8263980-a9c4-11eb-903a-309eade8483b.png)

In other words, after this change, those constants are left unused. Why?

Reply:
Aaaaah, I know where you are confused.

Michael, those constants are *not* mosaic constants. Take a closer look where this file is located :joy: